float32 coord_x
float32 coord_y
---
string room